languages/javalang/builder-renderer/generated-code/src/test/java/cleanup/DeleteEverythingIntegrationTest.java

```diff
@@ -0,0 +1,161 @@
+package cleanup;
+
+import com.commercetools.cart_discount.CartDiscountFixtures;
+import com.commercetools.category.CategoryFixtures;
+import com.commercetools.client.ApiRoot;
+import com.commercetools.discount_code.DiscountCodeFixtures;
+import com.commercetools.inventory.InventoryEntryFixtures;
+import com.commercetools.models.CartDiscount.CartDiscountPagedQueryResponse;
+import com.commercetools.models.Category.CategoryPagedQueryResponse;
+import com.commercetools.models.DiscountCode.DiscountCodePagedQueryResponse;
+import com.commercetools.models.Inventory.InventoryPagedQueryResponse;
+import com.commercetools.models.Product.ProductPagedQueryResponse;
+import com.commercetools.models.ProductDiscount.ProductDiscountPagedQueryResponse;
+import com.commercetools.models.ProductType.ProductTypePagedQueryResponse;
+import com.commercetools.models.Review.ReviewPagedQueryResponse;
+import com.commercetools.models.TaxCategory.TaxCategoryPagedQueryResponse;
+import com.commercetools.product.ProductFixtures;
+import com.commercetools.product_discount.ProductDiscountFixtures;
+import com.commercetools.product_type.ProductTypeFixtures;
+import com.commercetools.review.ReviewFixtures;
+import com.commercetools.tax_category.TaxCategoryFixtures;
+import com.commercetools.utils.CommercetoolsTestUtils;
+import org.junit.jupiter.api.Test;
+
+public class DeleteEverythingIntegrationTest {
```

Comment:
I know that we discussed this clean all tests, but it can be very dangerous if you accidentally run it on an important project. How about adding a javadoc that includes a warning to run this test?

Reply:
sure, but I think that the name "DeleteEverything" suggests danger :)